pkg/agent/datamodel/types.go

```diff
@@ -1675,6 +1683,7 @@ type NodeBootstrappingConfiguration struct {
 	DisableUnattendedUpgrades              bool
 	SSHStatus                              SSHStatus
 	DisableCustomData                      bool
+	OutboundType                           string
```

Comment:
this should also be `OutboundType` - RP will be able to use this type as well

Reply:
I think OutboundType should stay a string, because the pro of going with a string allows for RP to pass in a new string without us needing to add anything on the rp side. If we want to use the OutboundType it probably makes more since to go back to the iota

Reply:
I agree, was just confused since you were previously defining the type but not using it